ssg/templates.py

```diff
@@ -60,7 +71,7 @@ def load(self):
             self.langs.append(lang)
 
     def preprocess(self, parameters, lang):
-        # if no template.py file exists, skip this preprocessing part
+        # If no template.py file exists, skip this preprocessing part
```

Comment:
The comment is not needed, or you can make it literal by writing
```
if self.preprocessing_file_path is None:
    return None
other stuff 1
other stuff 2
```

Reply:
Sure.